src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -92,11 +96,6 @@ const mailInfo = (showOpt = true, address, officeName, faxNum) => {
   );
 };
 
-export const optionalDescription =
-  'These documents help us process this application faster.';
-export const requiredDescription =
-  'We require these documents in order to process this form.';
-
 // Return a boolean if there are any missing uploads where 'required'
```

Comment:
These were never imported anywhere, and now we will accept an optional argument that falls back to these values if need be.